demystifying_sql/demystifying_sql.md

```diff
@@ -0,0 +1,331 @@
+
+That said, knowing the specific "flavor" of SQL your database uses is especially useful when first getting started writing queries and troubleshooting errors.
+
+## SQL Queries
+
+A SQL **query** is essentially a question or request for data, written in a specific structure.  Let's take a closer look at how to compose a SQL query!
+
+<div class = "care">
+If you feel anxious when you see code, you have a great opportunity in this module.  We're going to give some simple examples of SQL code to help build your intuition about SQL.  You **won't** have to run any code and we're only going to barely scratch the surface of SQL **syntax** (supported commands and how to write them -- the grammar and vocabulary of SQL).  This will help build your intuition about what SQL is good at (picking out just the right data).
+</div>
+
+At a high level, we generally provide three pieces of information when constructing SQL "**queries**":
+
+ 1. The name of the **table(s)** where the data is stored.
+ 2. The **column name(s)** you want to look at from the table(s) you specified.  Want all the columns?  You can use an asterisk!
+ 3. Any **filtering condition(s)** you want to apply to your data pull.  This part is optional but is often used.
+
+You put these basic pieces of information together using the syntax shown below to create a SQL query:
+
+```sql
+SELECT _2_ FROM _1_ WHERE _3_;
+```
+
+For example, here are some sample queries, each of which take place on just a single table.
+
+```sql
+SELECT price FROM products WHERE product_type = "FRUIT";
+
+SELECT mpg, transmission_type FROM cars;
+
+SELECT * FROM patients WHERE age < 20;
+```
+
+### SELECT and FROM
+
+A **select statement** is used to specify which columns you would like to have returned as output from your SQL query.
+
+The basic components of a select statement are the `SELECT` and `FROM` keywords. The `FROM` keyword is used to specify the table or tables that hold the data you're interested in, and the `SELECT` keyword is used to provide a list of columns within those table(s) that you would like returned as output.  
+
+<div class = "hint">
+Many people choose write SQL keywords in all capital letters, but that's so that they stand out clearly in SQL code, not because it's a language requirement.  
+</div>
+
+**Select All Columns**
+
+If you would like to return **all** of the columns from the table(s) specified in your SQL query, you can use the `*` wild card character as shown in the example below.  You'll notice that we put a line break between the asterisk and the `FROM` keyword.  Spaces and line breaks, or **whitespace**, doesn't really matter for SQL.  You can run your code together on a single line, or (preferably) add intendation and spaces that help human readers understand your code more easily.
```

Comment:
Should it be "Spaces and line breaks, or **whitespace**, don't really matter for SQL."?

Reply:
yes!  subject verb agreement ftw!